### PR TITLE
update: Dragonfly GA limitations 

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
Updated Dragonfly documentation to include GA phase limitations:  Users, ACLs, and service configurations won’t auto-migrate from Redis to Dragonfly. 

[MA-2397](https://aiven.atlassian.net/browse/MA-2397)

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

[MA-2397]: https://aiven.atlassian.net/browse/MA-2397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by safa-topal — CHANGES_REQUESTED** [review]
Lgtm overall, left a comment on one statement which not correct yet but will be later. Also a nitpick.

**Review by safa-topal — APPROVED** [review]
Lgtm

File: docs/products/dragonfly/howto/migrate-aiven-redis-df-console.md [hunk 1]
```diff
@@ -38,12 +33,26 @@ The Aiven Console migration tool automatically uses connection details
...
+the most valuable features to you, guided by your feedback. During the initial phase of
+GA, automatic migration of Users, Access Control Lists (ACLs), and service configurations
+from Redis to Dragonfly will not be included.
+
+What this means:
+
+- **Users and ACLs:** Custom user settings and access controls must be manually
+configured in Dragonfly.
+- **Service configurations:** If you’ve customized your Redis service with
+specific settings, you must manually apply these settings to Dragonfly. Automatic
```
Comment (hoang-vo) [inline docs/products/dragonfly/howto/migrate-aiven-redis-df-console.md:48]: same here

**Comment by harshini-rangaswamy** [comment]
> we probably need to exclude ACL on DF

Updated content to remove ACL

File: static/includes/dragonfly-limitations.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+As Aiven for Dragonfly moves to General Availability (GA), the focus is on delivering
+the most valuable features to you, guided by your feedback. During the initial phase of
+GA, automatic migration of Users, Access Control Lists (ACLs), and service configurations
+from Redis to Dragonflyare not included.
+
+What this means:
+
+- **Service configurations:** If you’ve customized your Redis service with
+specific settings, you must manually apply these settings to Dragonfly. Automatic
+transfer of these custom configurations is not available during the initial phase of GA.
```
Comment (ArthurFlag) [inline static/includes/dragonfly-limitations.md:12]:
```suggestion
  specific settings, you must manually apply these settings to Dragonfly. Automatic
  transfer of these custom configurations is not available during the initial phase of GA.
```

**Review by ArthurFlag — COMMENTED** [review]
minor formatting issue and 🚀

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**